hooks/format-hook

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh  
+  
+if git rev-parse --verify HEAD >/dev/null 2>&1  
+then  
+against=HEAD  
+else  
+# Initial commit: diff against an empty tree object  
+against=$(git hash-object -t tree /dev/null)
+fi  
```

Comment:
i'd add a little bit of indentation here :yum:

Reply:
[6c181c1](https://github.com/iScsc/wiresmash/pull/30/commits/6c181c1fbb3053713f1a7adaf686bb8d45f7a4f2)